platform/security/platform-security-core-api/src/main/java/ddf/security/SystemHighAttributes.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.security;
+
+import java.io.Serializable;
+import java.util.List;
+
+/**
+ * This code is experimental. While this interface is functional and tested, it may change or be
+ * removed in a future version of the library.
+ */
+public interface SystemHighAttributes {
+
+  /**
+   * @return an {@link List<String>} of the {@link Serializable} values for the system high
+   *     attributeName, or {@List Collections#emptyList()} if the attributeName is not present.
+   */
+  List<Serializable> getValues(final String attributeName);
```

Comment:
❓  If there are no duplicated attributes, would it make sense to return a `Set<Serializable>` instead? Can we make that guarantee?

Reply:
Since order could matter, a `List` is probably safer here.

Reply:
I checked with @stustison that order should not matter. I updated the documentation to clarify this assumption and updated this method to return a `Set<String>` instead.